geode-membership/src/integrationTest/java/org/apache/geode/distributed/internal/membership/gms/membership/GMSJoinLeaveJUnitTest.java

```diff
@@ -848,6 +848,21 @@ private InstallViewMessage getInstallViewMessage(GMSMembershipView view, Object
     return installViewMessage;
   }
 
+  @Test
+  public void testNetworkPartitionDetectedWithNullView() throws Exception {
+    initMocks(true);
+    prepareAndInstallView(mockMembers[0], createMemberList(mockMembers[0], gmsJoinLeaveMemberId));
+
+    NetworkPartitionMessage message = new NetworkPartitionMessage();
+    message.setSender(gmsJoinLeaveMemberId);
+
+
+    // this creates a NPE that must be handled appropriately
+    gmsJoinLeave.currentView = null;
+    gmsJoinLeave.processNetworkPartitionMessage(message);
```

Comment:
Nitpick: maybe use `assertThatCode(() -> gmsJoinLeave.processNetworkPartitionMessage(message)).doesNotThrowAnyException();` to make the test more explicit instead?.